uiowa/uiowa
# Update drupal/coder to most recent version and fix new PHPCS errors

As part of the process of getting ready for Drupal 9, `drupal/coder` will be updated, which introduces some new PHPCS sniff rules. These new rules will cause builds to break. I discovered this in #2160. This PR is an effort to preemptively fix those issues before we have to, to simplify the future process and testing.

# To test
This PR shouldn't change any functionality in our custom code. Please review the [code changes](https://github.com/uiowa/uiowa/pull/2216/files) and test as you see fit.
